docs/install.md

```diff
@@ -1,77 +1,197 @@
-* Edit the variables in `scripts/start-wrapper.sh` and `scripts/start-turbinia-common.sh` as appropriate (please note that the `start-wrapper.sh` script has a `GOOGLE_APPLICATION_CREDENTIALS` environment var in the middle of the script that needs to be updated). 
-* In your worker VM, add a new custom metadata key `startup-script-url` pointing to `gs://$your_bucket/scripts/start-wrapper.sh`
-* Upon start, your VM should mount your GCS Bucket, and copy the start scripts into the home directory of the Turbinia user and will then start the Turbinia worker.
-* If you are running in GCP, you may also want to install [GCS FUSE](https://cloud.google.com/storage/docs/gcs-fuse).
-
-## Setup
-Turbinia can be run either in the cloud, or on local machines.  If you run Turbinia on local machines, it will still use cloud PubSub for the client to talk to the server, and for the server to talk to the worker nodes.
-
-### Local Setup
-Turbinia requires all worker nodes to have direct access to all Evidence data.  The easiest way to set this up on local machines is to have a NFS or SAN mounted on a common path on each worker.  All output should also be written to the common directory so that when Evidence is added back into Turbinia that the other worker nodes can process it.  Turbinia can also write output to GCS even when running locally (set the `GCS_OUTPUT_PATH` variable in the config).
-
-### Google Cloud Platform Setup 
-Turbinia can read Evidence from either cloud Persistent Disks, or from GCS objects.  Turbinia can also write output to GCS (set the `GCS_OUTPUT_PATH` variable in the config).  Note that you can operate multiple Turbinia instances within the same GCP Project as long as your careful to make sure your config (pubsub topics/subscriptions, output paths, etc) doesn't overlap.
-
-#### Persistent Disks
-Persistent disks should be the default when processing disks that come from the Cloud.  The account you run Turbinia as must have access to the Persistent Disks that you want to process.  If you add a GoogleCloudDisk Evidence type, the worker node will attach the disk automatically before it runs its tasks.
-
-#### GCS objects
-If you have raw disk images from physical machines or otherwise that you want to process in the cloud, you can process them directly from GCS.  This can be used when the images do not originate from VMs/containers that are running in the cloud.  You can potentially convert raw images to Persistent Disks, but that requires processing the image first, and so using GCS is recommended in this case.  Processing Evidence directly from GCS requires that you use [GCS FUSE](https://cloud.google.com/storage/docs/gcs-fuse) and mount your bucket at a common path on each worker node.  Once your GCS bucket is mounted, you can process these images as 'rawdisk' Evidence.
+# High-Level Setup
+
+Turbinia can be run either in the Google Cloud, or on local machines. If you run
```

Comment:
General comment for the doc:  I don't know much about markdown style, but is manually adding newlines into the text the idiomatic way to write these files?  There are line breaks in some strange places (e.g. in the middle of bash one-liners), but I guess it doesn't change their readability.

Reply:
WAI - used md linter.